src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -52,9 +52,14 @@ public CantGetEditAssetsException(String message, Throwable cause) {
         }
     }
 
-    public EditProtocol(ResourceIds resourceIds, ImageStore imageStore) {
+    public interface OnErrorListener {
```

Comment:
OnErrorListener should probably be in ViewVisitor now, so we don't have a circular dependency